lib/load.php

```diff
@@ -103,7 +107,7 @@ function gutenberg_is_experiment_enabled( $name ) {
 require_once __DIR__ . '/compat/wordpress-6.3/kses.php';
 
 // Experimental features.
-require __DIR__ . '/experimental/behaviors.php';
+remove_action( 'plugins_loaded', '_wp_theme_json_webfonts_handler' ); // Turns off WP 6.0's stopgap handler for Webfonts API.
```

Comment:
Why are we adding this? Seems unrelated to behaviors unless I'm missing something? 🤔

Reply:
It is indeed unrelated 🤔 . We should remove it, as it seems that we took that from a rebase from trunk.

Reply:
ok, I ll remove it!

Reply:
done [d9048d2](https://github.com/WordPress/gutenberg/pull/52370/commits/d9048d253482a50b7ae8b1cf8cd66f418ced5015)